src/VisualStudio/IntegrationTest/TestUtilities/InProcess/ChangeSignatureDialog_InProc.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls.Primitives;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Microsoft.VisualStudio.LanguageServices.Implementation.ChangeSignature;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    internal class ChangeSignatureDialog_InProc : InProcComponent
+    {
+        private ChangeSignatureDialog_InProc()
+        {
+        }
+
+        public static ChangeSignatureDialog_InProc Create()
+            => new ChangeSignatureDialog_InProc();
+
+        public void VerifyOpen()
```

Comment:
`VerifyOpen` makes it seem like it this would just check if it is currently open instead of waiting for it to be open.

Reply:
💭 Interesting. However, I'll leave this for now since this behavior didn't change as part of this pull request.